org.pegasus_frontend.Pegasus.yml

```diff
@@ -0,0 +1,28 @@
+app-id: org.pegasus_frontend.Pegasus
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: pegasus-fe
+finish-args:
+  - --device=dri
+  - --filesystem=home:ro
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+
+modules:
+  - name: pegasus-frontend
+    buildsystem: qmake
+    builddir: true
+    config-opts:
+      - USE_SDL_GAMEPAD=1
+      - INSTALL_BINDIR=/app/bin
+      - INSTALL_ICONDIR=/app/share/icons/hicolor/128x128/apps
```

Comment:
the metainfo requires 128 & 64 sizes.

Reply:
You mean both 128 and 64? I see, will follow up with a patch.